src/ProgressOnderwijsUtils/Data/ParameterizedSqlExtensions.cs

```diff
@@ -50,4 +50,14 @@ public static ParameterizedSql ConcatenateSql(this IEnumerable<ParameterizedSql>
         }
         return new SeveralSqlFragments(builder.ToArray()).BuildableToQuery();
     }
+
+    [Pure]
+    public static ParameterizedSql In<T>(this ParameterizedSql query, T[] set)
```

Comment:
De set zou ook wel IEnumerable type kunnen hebben volgens mij?

Reply:
Sure.

Reply:
De api is wel een beetje gek, want als je dit gebruikt in een grotere query zal die "query" hier niet heel logisch lijken, toch?

Reply:
We zouden ook iets als dit kunnen kiezen:

```C#
public static ParameterizedSql AsUnrolledSqlInExpression(this T[] set)
```

...maar dat levert wel potentieel wat namespace pollution op op een vrij veelgebruikt type.